shared/src/business/useCases/sealCaseContactAddressInteractor.test.js

```diff
@@ -39,6 +39,30 @@ describe('sealCaseContactAddressInteractor', () => {
     ).rejects.toThrow('Cannot seal contact');
   });
 
+  it('should default otherFilers and otherPetitioners to an empty array if they do not exist on the case', async () => {
+    const caseWithoutOthers = {
+      ...MOCK_CASE_WITH_SECONDARY_OTHERS,
+      otherFilers: null,
+      otherPetitioners: null,
+    };
+
+    applicationContext
+      .getPersistenceGateway()
+      .getCaseByDocketNumber.mockReturnValue(caseWithoutOthers);
+
+    applicationContext.getCurrentUser.mockReturnValue({
+      role: ROLES.docketClerk,
+      userId: 'docketClerk',
+    });
+    await expect(
+      sealCaseContactAddressInteractor({
+        applicationContext,
+        contactId: '23-skidoo',
+        docketNumber: MOCK_CASE.docketNumber,
+      }),
+    ).rejects.toThrow('Cannot seal contact');
```

Comment:
I'm not understanding the connection between the description of this test and the expectation for an error being thrown here - is an error thrown because of the empty arrays for `otherFilers`/`otherPetitioners`?

Reply:
Updated!